cfme/tests/networks/test_sdn_navigation.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from utils import testgen
+from cfme.cloud.provider.ec2 import EC2Provider
+from cfme.cloud.provider.azure import AzureProvider
+from cfme.cloud.provider.openstack import OpenStackProvider
+from utils.appliance.implementations.ui import navigate_to
+from cfme.networks.provider import NetworkProvider
+
+
+pytest_generate_tests = testgen.generate(
+    classes=[EC2Provider, AzureProvider, OpenStackProvider], scope='module')
+pytestmark = pytest.mark.usefixtures('setup_provider')
+
+
+@pytest.mark.parametrize("tested_part", ["Cloud Subnets", "Cloud Networks", "Network Routers",
+                         "Security Groups", "Floating IPs", "Network Ports", "Load Balancers"])
+def test_provider_relationships_navigation(provider, tested_part):
+    view = navigate_to(provider, 'Details')
+    net_prov_name = view.contents.relationships.get_text_of('Network Manager')
+
+    network_provider = NetworkProvider(name=net_prov_name)
+
+    view = navigate_to(network_provider, 'Details')
+    value = view.contents.relationships.get_text_of(tested_part)
+    if value != "0":
+        tested_view = navigate_to(network_provider, tested_part.replace(' ', ''))
```

Comment:
No, because strip method removes characters only at the beginning and end of string